spec/queries/matched_for_notification_spec.rb

```diff
@@ -1,73 +1,177 @@
 RSpec.describe MatchedForNotification do
   describe "#call" do
     before do
-      @list1 = create(:subscriber_list, tags: {
+      @subscriber_list1 = create(:subscriber_list, tags: {
+          topics: { any: ["oil-and-gas/licensing"] }, organisations: { any: ["environment-agency", "hm-revenue-customs"] }
+      })
+
+      @subscriber_list2 = create(:subscriber_list, tags: {
+          topics: { any: ["business-tax/vat", "oil-and-gas/licensing"] }
+      })
+
+      @subscriber_list3 = create(:subscriber_list, links: { topics: { any: ["uuid-123"] }, policies: { any: ["uuid-888"] } })
+
+      @subscriber_list4 = create(:subscriber_list,
+                                 links: { topics: { any: ["uuid-123"] } },
+                                 tags: {
+                                     topics: { any: ["environmental-management/boating"] },
+                                 })
+
+      @subscriber_list5 = create(:subscriber_list, links: { topics: { all: ["uuid-123", "uuid-234"] } })
+
+      @subscriber_list6 = create(:subscriber_list, links: { topics: { all: ["uuid-345", "uuid-456"], any: ["uuid-567", "uuid-678"] } })
+
+      @subscriber_list7 = create(:subscriber_list, links: { topics:
+                                                                                                  {
+                                                                                                      all: ["uuid-345", "uuid-456"]
+                                                                                                  },
+                                                                                              policies:
+                                                                                                  {
+                                                                                                      all: ["uuid-567", "uuid-678"]
+                                                                                                  } })
```

Comment:
The formatting seems quite off here - can it be fixed up? Difficult to read.

Reply:
Yup. Moral of the story: the linter can fix offences, but don't expect it to make things easier to read!